pkg/manager/servicesLeader.go

```diff
@@ -24,6 +25,11 @@ func (sm *Manager) startServicesWatchForLeaderElection(ctx context.Context) erro
 
 	for _, instance := range sm.serviceInstances {
 		for _, cluster := range instance.clusters {
+			err = cluster.Network.DeleteRoute()
```

Comment:
does the deletion of routes need checking if we're using routing table mode before attempting to delete?

Reply:
I think it's not. Just added that mostly to be sure and get rid of the linter issue, but deleted that just now.